src/main/java/roomescape/controller/AdminWebController.java

```diff
@@ -0,0 +1,28 @@
+package roomescape.controller;
+
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.GetMapping;
+
+@Controller
+public class AdminWebController {
+
+    @GetMapping("/admin")
+    public String getAdminPage() {
+        return "/admin/index";
+    }
```

Comment:
```suggestion
@Controller
@RequestMapping("/admin")
public class AdminWebController {

    @GetMapping
    public String getAdminPage() {
        return "/admin/index";
    }
```
하나의 controller 에서 공통으로 사용되는 uri 와 `RequestMapping`은 위로 추출해보는건 어떨까요?

Reply:
공통 uri 분리! 적용해보았습니다. 
정리하다 보니 하나의 컨트롤러에도 같은 uri prefix로 시작하는 엔드포인트들이 보여서 
컨트롤러의 분리도 함께 진행해주었습니다 😀